core/hw/sh4/dyna/driver.cpp

```diff
@@ -38,6 +38,7 @@ uintptr_t cc_rx_offset;
 u32 LastAddr;
 u32 LastAddr_min;
 u32* emit_ptr=0;
+t_mainloop_fn mainloop_function;
```

Comment:
I'd name this `MainLoopFN_t` or some such to fit with the original nullDC naming

Reply:
`MainloopFp_t* whatever`